src/model/Field.tsx

```diff
@@ -5,7 +5,7 @@ export interface FieldProps {
     label: string;
     description?: string;
     mandatory: 'mandatory' | 'optional' | 'recommended';
-    cardinality: 'single' | 'multiple';
+    requirementType: 'single' | 'multiple';
```

Comment:
This is called "multipliticy"
```suggestion
    multiplicity: 'single' | 'multiple';
```